pal/input/web/accelerometer-input.ts

```diff
@@ -120,12 +121,13 @@ export class AccelerometerInputSource {
 
     public start (): void {
         // for iOS 13+, safari
-        if (window.DeviceMotionEvent && typeof DeviceMotionEvent.requestPermission === 'function') {
-            DeviceMotionEvent.requestPermission().then((response) => {
+        // NOTE: since TS 4.4, requestPermission is not defined in class DeviceMotionEvent, this should be a breaking change in TS.
```

Comment:
```suggestion
        // NOTE: since TS 4.4, `requestPermission` is not defined in class DeviceMotionEvent in `lib.dom.d.ts`, this should be a breaking change in TS.
        // Accessing the `requestPermission` would emit a type error, so we assert `DeviceMotionEvent` as any type to skip the TS type checking.
```